### DISCUSSION TITLE
合集整理功能优化

### DISCUSSION THREAD

**Discussion by nathandingx** [jxxghp/MoviePilot#3434] [讨论]
### 当前程序版本

最新版本

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

前些日子下载了一部系列作品【动漫剧集+电影+OVA的整合包】，整理的时候程序只帮忙整理了剧集，但同一个下载文件夹内的电影、OVA没有整理，希望相关功能能得到优化（PS：因作品影响力较大，所以出了动漫、电影、OVA，并非故意找茬不单独下载，单独下载我害怕漏掉没下载）

**Comment by MapleRedbean** [comment]
我发现有些OVA不是识别，然后一些很多季合集的也不好识别。只会识别第一季
